test/s3select_test.cpp

```diff
@@ -629,3 +629,84 @@ TEST(TestS3selectFunctions, modzero)
     ASSERT_EQ(s3select_result, std::string("0,\n"));
   }
 
+  TEST(TestS3selectFunctions, null)
+{
+    s3select s3select_syntax;
+    const std::string input_query = "select 1 + null from stdin;";
```

Comment:
as discussed , this currently could not work properly, need a change in syntax-semantic parsing.
add unit-test for operators.
such as 
TEST(TestS3SElect, float_compare_operator)
TEST(TestS3SElect, arithmetic_operator)